.pre-commit-config.yaml

```diff
@@ -48,7 +48,7 @@ repos:
         exclude: security-policies.*
 
   - repo: https://github.com/psf/black
-    rev: 23.12.0
+    rev: 23.12.1
```

Comment:
@orestisfl should it only update minors?

Reply:
I don't think so, given that patch versions are usually bugfixes